src/HDFWriterModule.cpp

```diff
@@ -12,11 +12,7 @@ std::map<std::string, HDFWriterModuleRegistry::ModuleFactory> &getFactories() {
 HDFWriterModuleRegistry::ModuleFactory &find(std::string const &key) {
   static HDFWriterModuleRegistry::ModuleFactory empty;
```

Comment:
```suggestion
```

If `find` now signals "not found' via an exception instead of the previous "empty value" then you can remove `empty` as well.